doc/22-selinux.md

```diff
@@ -104,7 +104,7 @@ SELinux is based on the least level of access required for a service to run. Usi
 
 **icinga2_can_connect_all** 
 
-Having this boolean enabled allows icinga2 to connect to all ports. This can be necessary if you use features which connect to unconfined services.
+Having this boolean enabled allows icinga2 to connect to all ports. This can be necessary if you use features which connect to unconfined services, e.g. the [influxdb writer](14-features.md#influxdb-writer).
```

Comment:
Two things about wording. First there are three paragraphs following the same style and always starting with 'Having this boolean enabled...', that's annoying to read especially since the header is "Booleans".
Also I'm not a fan of the way we use e.g. and i.e. in the documentation, i.e. we use it too much (and sometimes wrong).
In this case 'for example' is clearer and we need not be frugal with our bytes ^_^

Reply:
You are absolutely right, I will change the abbreviation and try to find another phrase or two for the wording.